### ISSUE TITLE
README.org issue

### ISSUE THREAD

**Issue by kamysheblid** [atlas-engineer/next#271]
In section "Run From Source" between 1 and 2 was needed this instruction to get mine to work:

(ql:quickload "trivial-features")

I had a fresh install of sbcl and quickload, but when I followed the instruction it kept giving me an error about trivial-features. Eventually I looked at the quickload site and tried the command above and it fixed my problem.

After fixing README.org this is what I have:
```
1. ~$LISP~ to create a new Lisp REPL (replace ~$LISP~ with ~sbcl~ or any
   supported Common Lisp compiler).
2. Execute ~(ql:quickload "trivial-features")~
3. Execute ~(require :asdf)~ if ASDF is not already loaded.
4. Execute ~(asdf:load-asd "/full/path/to/next.asd")~ to load the Next
   system definition (you can not use relative pathnames).
5. Execute ~(ql:quickload :next)~ to load the Next system into your
   Lisp image.
6. Start the platform port and make sure it will be found by Next as described
   in the previous section.
7. Execute ~(next:start)~ to open your first Next window.
```
Or maybe it might be better to just put that it in "Manually Installing Quicklisp" since it only needs to  be run once.

**Comment by alexanderbarbosa** [comment]
You can just `make all`, and Next will take care of everything to be correctly installed.

**Comment by Ambrevar** [comment]
Good catch, fixed in 6f73ea21735645a20d04a9721b53ce5cf487f292.  Thanks!
